Library/Output Templates/fantasy_grounds_pc_template.xml

```diff
@@ -0,0 +1,250 @@
+          <ammo type="number">@AMMO</ammo>
+          <rangedmodelist>@ATTACK_MODES_LOOP_START
+            <id-@ID>
+              <name type="string">@USAGE</name>
+              <level type="number">@LEVEL</level>
+              <damage type="string">@DAMAGE</damage>
+              <acc type="number">@ACCURACY</acc>
+              <range type="string">@RANGE</range>
+              <rof type="string">@ROF</rof>
+              <shots type="string">@SHOTS</shots>
+              <rcl type="number">@RECOIL</rcl>
+            </id-@ID>@ATTACK_MODES_LOOP_END
+          </rangedmodelist>
+        </id-@ID>@HIERARCHICAL_RANGED_LOOP_END
+	</rangedcombatlist>
+
+      <protectionlist>
+        @HIT_LOCATION_LOOP_START
+        <id-@ID>
+          <location type="string">@WHERE</location>   
+          <dr type="string">@DR</dr>
+          <db type="string">@PENALTY</db>
+          <name type="string">@EQUIPMENT</name>
+  	    <text type="formattedtext">@EQUIPMENT_FORMATTED</text>
+        </id-@ID>@HIT_LOCATION_LOOP_END
+      </protectionlist>
+    </combat>
+
+    @ENCUMBRANCE_LOOP_START
+    <enc@LEVEL_ONLY@dodge type="number">@DODGE</enc@LEVEL_ONLY@dodge>@ENCUMBRANCE_LOOP_END
+
+    <encumbrance>@ENCUMBRANCE_LOOP_START
+      <enc_@LEVEL_ONLY type="number">@CURRENT_MARKER_1</enc_@LEVEL_ONLY>@ENCUMBRANCE_LOOP_END
+      @ENCUMBRANCE_LOOP_START
+      <enc@LEVEL_ONLY@_weight type="string">@MAX_LOAD</enc@LEVEL_ONLY@_weight>@ENCUMBRANCE_LOOP_END
+      @ENCUMBRANCE_LOOP_START
+      <enc@LEVEL_ONLY@_move type="string">@MOVE</enc@LEVEL_ONLY@_move>@ENCUMBRANCE_LOOP_END
+      @ENCUMBRANCE_LOOP_START
+      <enc@LEVEL_ONLY@_dodge type="number">@DODGE</enc@LEVEL_ONLY@_dodge>@ENCUMBRANCE_LOOP_END
+    </encumbrance>
+
+    <traits>
+      <race type="string">@RACE</race>
+      <height type="string">@HEIGHT</height>
+      <weight type="string">@WEIGHT</weight>
+      <age type="string">@AGE</age>
+      <appearance type="string">@GENDER, Eyes: @EYES, Hair: @HAIR, Skin: @SKIN</appearance>
+      <sizemodifier type="string">@SIZE</sizemodifier>
+
+      @EXCLUDE_CATEGORIES_POWER_OTHER
+      <adslist>@ADVANTAGES_ALL_LOOP_START
+        <id-@ID>
+          <name type="string">@DESCRIPTION_PRIMARY</name>
+          <points type="number">@POINTS</points>
+          <text type="string">@DESCRIPTION_MODIFIER_NOTES @DESCRIPTION_NOTES_PAREN @REF</text>
+        </id-@ID>@ADVANTAGES_ALL_LOOP_END
+      </adslist>
+
+      @EXCLUDE_CATEGORIES_POWER_OTHER
+      <disadslist>@DISADVANTAGES_ALL_LOOP_START
+        <id-@ID>
+          <name type="string">@DESCRIPTION_PRIMARY</name>
+          <points type="number">@POINTS</points>
+          <text type="string">@DESCRIPTION_MODIFIER_NOTES @DESCRIPTION_NOTES_PAREN @REF</text>
+        </id-@ID>@DISADVANTAGES_ALL_LOOP_END
+      </disadslist>
+
+      <culturalfamiliaritylist>@CULTURAL_FAMILIARITIES_LOOP_START
+        <id-@ID>
+          <name type = "string">@DESCRIPTION_PRIMARY</name>
```

Comment:
For consistency, remove the spaces on either side of the = sign here and elsewhere

Reply:
Done.   Also, updated to include description notes/modifier notes and page references for all items that support them.

Reply:
Still seeing several things like `<name type = "string">` that should be `<name type="string">` for consistency.

Reply:
Got it that time.   I didn't know I needed the "-a" option.